HelpSource/Classes/Main.schelp

```diff
@@ -51,7 +51,7 @@ Called after the class library has been compiled.
 discussion::
 This calls the superclass' startup, which among other things initializes the link::Classes/AppClock:: and the top-level link::Classes/Environment::.
 
-Main's startup then stores Server.default in the interpreter variable s, sets the platform default's link::Classes/GUI:: kit, calls a link::Classes/Platform:: specific startup method (for example OSXPlatform's startup opens the server windows), and finally invokes StartUp.run.
+Main's startup then stores Server.default in the interpreter variable s, sets the platform default's link::Classes/GUI:: kit, calls a link::Classes/Platform:: specific startup method (for example macOSPlatform's startup opens the server windows), and finally invokes StartUp.run.
```

Comment:
Thought I had caught all the accidental changes of `OSXPlatform`, thanks for catching that one! Eventually the class should be renamed `MacOSPlatform`, but I wasn't intending to tackle that here.